lib/src/player_with_controls.dart

```diff
@@ -52,14 +57,17 @@ class PlayerWithControls extends StatelessWidget {
                 PlayerNotifier notifier,
                 Widget? widget,
               ) =>
-                  AnimatedOpacity(
-                opacity: notifier.hideStuff ? 0.0 : 0.8,
-                duration: const Duration(
-                  milliseconds: 250,
-                ),
-                child: Container(
-                  decoration: const BoxDecoration(color: Colors.black54),
-                  child: Container(),
+                  Visibility(
```

Comment:
Just curious as to why the `Visibility` widget above the `AnimatedOpacity` one. Is it because the latter interferes with `InteractiveViewer`?

Reply:
That's correct. Even with opacity at zero the gestures will not be picked by the InteractiveViewer. So if we hide the AnimatedOpacity it would work correctly.